source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Deploying oVirt and Gluster Hyperconverged
+---
+
+# Chapter 2: Deploying oVirt and Gluster Hyperconverged
+
+## Pre-requisites
+
+* You must have 3 freshly installed Enterprise Linux 7 hosts.
+
+* You must have atleast 2 interfaces on each of the hosts, so that the frontend and backend traffic can be separated out. Having only one network will cause the engine monitoring, client traffic, gluster I/O traffic to all run together and interfere each other. To segregate the backend network, the gluster cluster is formed using the backend network addresses, and the nodes are added to the engine using the frontend network address.
+
+* You must have a fully qualified domain name prepared for your Engine and the host. Forward and reverse lookup records must both be set in the DNS. The Engine should use the same subnet as the management network.
+
+* You must have configured passwordless ssh between the first host to itself and the other 2 hosts. This is needed for gdeploy to configure the 3 nodes. gdeploy uses Ansible playbooks and the ability to remotely execute commands is a pre-requisite.
+
+## Deploying on Enterprise Linux Hosts
+
+### Installing the Required Packages
+
+**Installing the packages on the first host**
+
+1. Subscribe to ovirt repos from http://resources.ovirt.org/pub/yum-repo/
+   For instance, to subscribe to oVirt 4.1 repo,
+
+        # yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+
+2. Install gdeploy and cockpit-ovirt that will provide a UI for the installation. gdeploy is a wrapper tool around Ansible that helps to setup gluster volumes.
+      
+        # yum install gdeploy cockpit-ovirt
+
+3. Install the oVirt Engine Virtual Appliance package for the Engine virtual machine installation:
+
+        # yum install ovirt-engine-appliance
+
+
+### Setting up the hyperconverged environment
+
+Steps for installing are detailed in this [blog post](/blog/2017/04/up-and-running-with-ovirt-4.1-and-gluster-storage/).
```

Comment:
Few review comments on the blog post.
1) screenshot for the volume types should be changed as engine volume is also indicated as arbiter, where as we would want engine volume to be of type replica for HA.
2)There is a screenshot which displays the engine brick as 'thinp' where as it should be thick.
3) Remove the step of adding disable-multipath.sh script as this is already present with the latest cockpit builds.
4) screenshot which represents to import hosts after successful hosted-engine deployment should be removed as additional hosts should be added after adding the first storage domain and once hosted_storage and HostedEngine vm is imported into the UI.
5) screenshot for the 'New Logical Network' should be changed as 'VM network' is selected. 'VM network' should be unchecked while creating 'New Logical Network'
6)  It would be good if we can interchange the Manage Network and SetupHostNetworks screenshot as Manage network should be done prior to SetupHostNetworks.
7)